examples/nitclk/synchronize-tclk.py

```diff
@@ -0,0 +1,164 @@
+# Demonstrates how to configure multiple NI Arbitrary Waveform Generators to generate synchronized waveforms with TClk
+# 
+# The gRPC API is built from the C API. NI-FGEN documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niFgen\documentation\English\SigGenHelp.chm
+#
+# NI-TClk documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niTClk\documentation\English\nitclk.chm
+#
+# Getting Started:
+#
+# To run this example, install "NI-FGEN Driver" on the server machine.
+# Link : https://www.ni.com/en-us/support/downloads/drivers/download.ni-fgen.html
+#
+# For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC Client" wiki page.
+# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+#
+# Running from command line:
+#
+# Server machine's IP address, port number, and multiple comma separated resource names can be passed as separate command line arguments.
+#   > python synchronize-tclk.py <server_address> <port_number> <resource_name>
+# This example doesn't work on simulation, so these arguments are mandatory, not optional
```

Comment:
Change the comment to - This example is not supported in simulation mode, hence these arguments are mandatory